Project.toml

```diff
@@ -0,0 +1,10 @@
+name = "Roots"
+uuid = "f2b01f46-fcfa-551c-844a-d8ac1e96c665"
+
+[deps]
+Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
+Statistics = "1c261c3c-9a38-5c8c-b229-00278e4435fb"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
```

Comment:
Test doesn't need to be a direct dependency (unless it is actually used as one, but that would be weird). Instead, do
```toml
[extras]
Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
# any other test-only dependencies here

[targets]
test = ["Test"]  # and any others
```